src/App.jsx

```diff
@@ -1,6 +1,21 @@
 import React from 'react';
+import { Person } from './Person';
 import './App.scss';
 
+export const App = () => (
+  <div className="App">
+    <Person
+      person={misha}
+    />
```

Comment:
There is no need to split it into several lines when there are fewer than 3 props
```suggestion
    <Person person={misha} />
```